desktop-src/shell/notification-area.md

```diff
@@ -116,9 +116,11 @@ Note that this call to [**Shell\_NotifyIcon**](/windows/desktop/api/Shellapi/nf-
 
 ### Define the Notification Look and Contents
 
-A notification is a special type of balloon tooltip control. It contains a title, body text, and an icon. Like a window, it has a **Close** button in its upper right corner. It also contains a **Options** button that opens the Notification Area Icons item in the Control Panel, which allows the user to show or hide the icon or show only notifications without an icon.
+A notification is a special type of balloon tooltip control. It contains a title, body text, and an icon. Like a window, it has a **Close** button in its upper right corner. In versions prior to Windows 10 it also contained a **Options** button that opens the Notification Area Icons item in the Control Panel, which allows the user to show or hide the icon or show only notifications without an icon. In Windows 10 and newer it displays at the left side of the close icon a menu icon which allows the user to turn off such notifications.
```

Comment:
In the other files, referencing newer versions of Windows is phrased as:
"As of Windows 10 and newer"
rather than
"In Windows 10 and newer"
And those other instances all put a comma after "newer".

I would also add a comma after "In versions prior to Windows 10" in this paragraph.

Reply:
So, the official style guide says that you should use `and later` and not `or newer`.

That's something I didn't knew when I wrote this.

Reply:
That's a good point. These docs may have been written before that standard. Updating the `or newer` instances to `and later` would be a good idea. I think there are several of them in these files.